internal/search/types.go

```diff
@@ -251,42 +251,34 @@ type RepoOptions struct {
 
 func (op *RepoOptions) String() string {
 	var b strings.Builder
-	if len(op.RepoFilters) == 0 {
-		b.WriteString("r=[]")
-	}
-	for i, r := range op.RepoFilters {
-		if i != 0 {
-			b.WriteByte(' ')
+	b.WriteString("RepoFilters: ")
+	if len(op.RepoFilters) > 0 {
+		for i, r := range op.RepoFilters {
+			if i != 0 {
+				b.WriteByte(' ')
+			}
+			b.WriteString(strconv.Quote(r))
 		}
-		b.WriteString(strconv.Quote(r))
+	} else {
+		b.WriteString("[]")
 	}
+	b.WriteString("\n")
 
 	if len(op.MinusRepoFilters) > 0 {
-		_, _ = fmt.Fprintf(&b, " -r=%v", op.MinusRepoFilters)
-	}
-	if op.CommitAfter != "" {
-		_, _ = fmt.Fprintf(&b, " CommitAfter=%q", op.CommitAfter)
+		_, _ = fmt.Fprintf(&b, "MinusRepoFilters: %v\n", op.MinusRepoFilters)
```

Comment:
```suggestion
		fmt.Fprintf(&b, "MinusRepoFilters: %v\n", op.MinusRepoFilters)
```

Haven't tested, but I think you can do this? Seems allowed for `WriteString` anyway

Reply:
Not the most aesthetically pleasing, but I do prefer to explicitly indicate ignoring of return values

Reply:
What's the rationale for not doing this for `b.WriteString` though? We can do this, but I prefer we are consistent about ignoring return values not on a case-by-case basis. My sense is that `fmt.Fprintf` falls into the same bucket of ignoring `b.WriteString` return values (little utility). The overwhelmingly popular convention seems to be to ignore `fmt.Fprintf` return values: https://sourcegraph.com/search?q=context:global+lang:go+Fprintf&patternType=literal. In fact, there isn't a single Go result for [`_, _ := fmt.Fprintf`](https://sourcegraph.com/search?q=context:global+lang:go+_%2C+_+:%3D+fmt.Fprintf&patternType=literal) across all the Go repos we index (which is quite something! didn't expect that!).

Reply:
> In fact, there isn't a single Go result for `_, _ := fmt.Fprintf` across all the Go repos we index

Probably because that's a compile error 🙂 [Try](https://sourcegraph.com/search?q=context:global+lang:go+_%2C+_+%3D+%28fmt%5C.%29%3FFprintf&patternType=regexp) `_, _ = fmt.Fprintf`

Reply:
[Compute coming in clutch](https://sourcegraph.com/notebooks/Tm90ZWJvb2s6ODU2)

Reply:
Ah shoot 🤦 thanks. Now I'm tabulating it 😆 https://codestat.dev/explorer?q=lang%3Ago%20content%3Aoutput((_%2C%20_%20%3D)%3F%20(fmt.Fprintf)%20-%3E%20%241%20%242)%20%20count%3Aall&t=chart&dp=30

Reply:
lol we thought to do the same!1

Reply:
![Screen Shot 2022-05-04 at 8 10 04 PM](https://user-images.githubusercontent.com/888624/166858425-523f517a-d450-4756-82b9-af60953f8be1.png)

(I think it ran over the 60 second timeout though)

Reply:
yeah it times out (redid it in notebooks and fixed my typo with the space)

https://sourcegraph.com/notebooks/Tm90ZWJvb2s6ODU2 

![Screen Shot 2022-05-04 at 8 14 24 PM](https://user-images.githubusercontent.com/888624/166858733-0b7aa98a-45d9-4f95-90a7-f3edffc54f71.png)

Reply:
(side note: this would be a really good query example to optimize pipeline overhead with)

Reply:
My rationale was that because `WriteString()` always returns a nil error, but `fmt.Fprintf()` isn't guaranteed to return a nil error, I wanted to clearly indicate we're ignoring a possibly non-nil error value. In my mind it adds a tiny bit of clarity, but yes, little utility.

However, I didn't think it'd be that big of an imbalance. It seems like the people have spoken on this one. I'll prefer to go with the convention and make a quick PR to ignore the return values of `fmt.Fprintf()` for consistency here.

Reply:
> My rationale was that because WriteString() always returns a nil error, but fmt.Fprintf() isn't guaranteed to return a nil error,

Hey fair point!

> However, I didn't think it'd be that big of an imbalance. It seems like the people have spoken on this one.

Part of this is just me understanding the choice, not a big deal, it just scanned a bit awkward reading the code. Sometimes these discussions lead to deciding on conventions that go the other way once there's a better shared understanding.